website/docs/d/elasticsearch_domain.html.markdown

```diff
@@ -0,0 +1,77 @@
+---
```

Comment:
In addition to this new file, you'll also want to add a link to `aws.erb` in the `website` dir - those should be kept in alphabetical order but should be pretty straightforward to figure out 🙂